examples/qatzip-envoy-conf.yaml

```diff
@@ -39,11 +39,17 @@ static_resources:
               - match:
                   prefix: "/"
                 route:
+                  timeout: 300s
                   cluster: cluster_0
           http_filters:
-          - name: envoy.qatzip
-            config:
-              compressor:
-                content_length: 48
+          - name: envoy.filters.http.compressor
+            typed_config:
+              "@type": type.googleapis.com/envoy.extensions.filters.http.compressor.v3.Compressor
+              content_length: 10
+              compressor_library:
+                name: testqat
+                typed_config:
+                  "@type": type.googleapis.com/qatzip.compressor.Qatzip
+                  hardware_buffer_size: sz_8k
```

Comment:
Why not?

Reply:
Maybe put the question in other way: is the user expected to correspond this value to something in the kernel or libqat? Or is string `sz_8k` mentioned in the HW docs? Otherwise it seems to me that something like "8k" would sound more natural. Or how the user expected to choose which value to put here? Sorry, I'm a newbie in the QAT compression world. :-)

Reply:
I see. QATZip expects integer here. Yet this integer can be of 6 possible values only. That's why I resorted to Enum. I guess I tried to name the enum values as 8k, 16k,.. first, but then realized that literals should start with a letter. So I decided to add the `sz_` prefix to indicate it's an Enum explicitly and not to encourage users to try something like 8M.